bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcPanel.scala

```diff
@@ -47,6 +47,12 @@ sealed trait ItcPanel extends GridBagPanel {
   def display: Component
   def visibleFor(t: SPComponentType): Boolean
 
+  /** TODO: we need to display a list of asterism members and let the user select one. For now we
+    * just select the first target in the asterism.
+    */
+  def selectedTarget: Option[Target] =
+    Option(owner.getContextTargetEnv).map(_.getAsterism.allTargets.head)
+
```

Comment:
It's a good question how to handle this.  For GHOST dual-target I think the two targets need to be nearly the same magnitude since they are being observed by the same detector.